src/routes/project/components/TextEditor.js

```diff
@@ -0,0 +1,194 @@
+import React, { Component } from 'react';
+import Editor from 'draft-js-plugins-editor';
+import { EditorState, RichUtils, getVisibleSelectionRect, convertToRaw, convertFromRaw } from 'draft-js'
+import createCounterPlugin from 'draft-js-counter-plugin';
+
+const counterPlugin = createCounterPlugin();
+const { CharCounter, WordCounter, CustomCounter } = counterPlugin;
+const plugins = [counterPlugin];
+
+
+class StyleButton extends Component {
+  constructor() {
+    super();
+    this.onToggle = (e) => {
+      e.preventDefault();
+      this.props.onToggle(this.props.style);
+    };
+  }
+
+  render() {
+    let className = 'toolbarItem';
+    if (this.props.active) {
+      className += ' active';
+    }
+    return (
+      <span className={className} onMouseDown={this.onToggle}>
```

Comment:
I did some digging and certain items cannot be flexed, check this SO thread: https://stackoverflow.com/questions/35464067/flexbox-not-working-on-button-element-in-some-browsers